Makefile

```diff
@@ -45,6 +45,9 @@ VERSION = ${GIT_TAG}
 override LDFLAGS += -X ${PACKAGE}.gitTag=${GIT_TAG}
 endif
 
+KUBECTX          := $(shell kubectl config current-context)
+ALLOWED_KUBECTXS := docker-desktop|docker-for-desktop|minikube|k3s-default
```

Comment:
This doesn't support named `minikube` instances. For example, if a start `minikube` with `minikube start -p test` then the output of `kubectl config current-context` would be `test`.

Reply:
do you use a named instance?

Reply:
that said, I think we just need to decide on the correct test here, kubectx might not be it.

Reply:
what about this

```
kubectl config view --minify | grep 'server:' | grep -o localhost
```